array/README.md

```diff
@@ -428,6 +428,34 @@ Skips the given count of elements from the given array.
 ```
 <!-- prettier-ignore-end -->
 
+# slidingWindow
+
+Returns a new list composed of tuples of the given sliding window length of consecutive elements.
```

Comment:
Please use `array` wording as there are no `lists` actually but plain arrays here in JS.